Bio/PDB/PDBIO.py

```diff
@@ -294,7 +294,7 @@ def _get_atom_line(
             return _PQR_ATOM_FORMAT_STRING % args
 
     # Public methods
-    def save(self, file, select=_select, write_end=True, preserve_atom_numbering=False):
+    def save(self, file, select=_select, write_end=True, write_ter = True, preserve_atom_numbering=False):
```

Comment:
A technical point, this breaks backward compatibility (in case anyone is currently calling the method with four positional arguments). You should add any new optional argument at the end.

Reply:
sorry , got lost in git and github branches and tag, ended up just filling on browser,   anyone refactoring everything to:     def f(pos1, pos2, /, pos_or_kwd, *, kwd1, kwd2):